README.md

```diff
@@ -33,40 +33,142 @@ Clover Go supports two types of card readers: a magnetic stripe, EMV chip-and-si
+
+5. Click the **Market Listing** tab and then on **Preview in App Market**.
+![oauth5a](/images/oauth5a.png)
+
+   **Preview In App Market** opens the app preview page as your test merchant:
+![oauth5b](/images/oauth5b.png)
+
+6. If the app is not installed for your test merchant, click **Connect** and then **Accept** to install the app. If the app is installed, click **Open App**. Either of these steps will open a browser tab with a URL containing a **CODE** parameter: 
+
+   `https://sandbox.dev.clover.com/?merchant_id={MERCHANT_ID}1&employee_id={EMPLOYEE_ID}&client_id={CLIENT_ID}&code={CODE}`
+![oauth6](/images/oauth6.png)
+
+   Save the **CODE** value somewhere.
+
+7. Pass in the **App ID**, **App Secret**, and **CODE** you saved earlier into the following URL: 
+
+   `https://sandbox.dev.clover.com/oauth/token?client_id={APP_ID}&client_secret={APP_SECRET}&code={CODE}`
+
+   (do not include the curly braces).
+
+8. Visit that URL in your browser, and you should be provided with your access token 🎉. 
+
+   **Note**: If you get an “Unknown Client ID” message, check that you don’t include any spaces in the URL and visit the URL again.
+
+### Running the Sample App
+
+#### Prerequisites
+1. Clone the SDK, go into the Example folder and check out the branch for Swift 5:
+
+       git clone https://github.com/clover/remote-pay-ios-go.git
+       cd remote-pay-ios-go/Example
+       git checkout CloverGo_Swift5.0
+2. Ensure that the following dependencies in your `Podfile` are set to the right versions:
+
+       pod 'Starscream', :git => 'https://github.com/daltoniam/Starscream.git', :tag => '3.0.5'pod 'GoConnector', '3.3.5'
+
+3. Update your project dependencies by running `pod install`.
+
+##### Note:
+Set swift_version to 4.2 for your pods if you run into incompatibility issues, and make sure to check out the branch for Swift 4.2.
+
+#### Xcode Steps
+1. Close any current Xcode sessions and open `CloverConnector.xcworkspace`.
+2. Set your `Enable Bitcode` setting to **false** for following:
+
+   -- **Project Target** (CloverConnector > CloverConnector_Example > Build Options > Enable Bitcode = No)
+
+   -- **Pod CloverGoSDK** (Pods > CloverGoSDK > Build Settings > Build Options > Enable Bitcode = No)
+   ![xcode](/images/xcode.png)
+
+   -- **Pod GoConnector** (Pods > GoConnector > Build Settings > Build Options > Enable Bitcode = No)
+3. In `ViewController.swift`, set your **accessToken**, **apiKey**, and **secret** in the following code block:
+
+```
+   override func viewDidLoad() {
+        super.viewDidLoad()
+        PARAMETERS.accessToken = "Access token that you generated via the OAuth flow earlier"
+        PARAMETERS.apiKey = "Get this value from your DevRel representative"
+        PARAMETERS.secret = "Get this value from your DevRel representative"
+        if let savedEndpoint = UserDefaults.standard.string(forKey: WS_ENDPOINT) {
+            endpointTextField.text = savedEndpoint
+        }
+    }
+```
+
+### FAQ
```

Comment:
FAQ's?

Reply:
Also, should this be **## FAQ's** (a bigger title)?

Reply:
should be ok